soc/silabs/silabs_siwx91x/siwg917/Kconfig

```diff
@@ -3,3 +3,34 @@
 
 config SOC_SERIES_SIWG917
 	select SOC_FAMILY_SILABS_SIWX91X
+
+config WISECONNECT_NETWORK_STACK
+	bool
+	select CMSIS_RTOS_V2
+	select POLL
+	select DYNAMIC_THREAD
+	select THREAD_NAME
+	select THREAD_STACK_INFO
+	select THREAD_MONITOR
+	select INIT_STACKS
+
+if WISECONNECT_NETWORK_STACK
```

Comment:
goes in `Kconfig.defconfig` file

Reply:
done.

Reply:
not addressed

Reply:
I got fooled by `git rebase`. Anyway, fixed now.